### PR TITLE
Initial draft of the HHS hospitalizations ground truth indicator

### PR DESCRIPTION
**Pull request by krivard**
### Description
New indicator based on definition [here](https://github.com/reichlab/covid19-forecast-hub/blob/master/data-processed/README.md#hospitalizations). This indicator fetches the two admissions columns from the Epidata dataset `covid_hosp`, sums them, and returns the result.

Only available at state level for now.

### Changelog
- hhs_hosp directory and accoutrements
- nominal tests for date conversion

### Fixes 
- Fixes https://app.asana.com/0/0/1199529848478514/f

### CODE REVIEW

File: hhs_hosp/delphi_hhs/run.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    # get a list of state codes
+    all_states = pd.DataFrame(
+        { "state_code": ["{:02d}".format(x) for x in range(1, 72)] }
+    )
+    all_states =  mapper.add_geocode(all_states, "state_code", "state_id")
+    request_all_states = ",".join(all_states.state_id)
+
+    today = date.today()
+    month_ago = today - timedelta(days=14)
```
Comment (rumackaaron) [inline hhs_hosp/delphi_hhs/run.py:42]: Maybe this is a bad variable name. Why do we get data for the last 14 days? Revisioning?

File: hhs_hosp/setup.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    version="0.1.0",
+    description="SHORT DESCRIPTION",
+    author="",
+    author_email="",
+    url="https://github.com/cmu-delphi/covidcast-indicators",
+    install_requires=required,
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Programming Language :: Python :: 3.7",
```
Comment (rumackaaron) [inline hhs_hosp/setup.py:27]: The makefile uses Python 3.8. Probably not a big deal

File: hhs_hosp/delphi_hhs/run.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    return x.apply(lambda x: datetime.strptime(str(x), "%Y%m%d") - timedelta(days=1))
+
+def run_module():
+    """Generate ground truth HHS hospitalization data."""
+    params = read_params()
+    mapper = GeoMapper()
+
+    # get a list of state codes
+    all_states = pd.DataFrame(
+        { "state_code": ["{:02d}".format(x) for x in range(1, 72)] }
```
Comment (rumackaaron) [inline hhs_hosp/delphi_hhs/run.py:36]: Is there a simple way to get only state codes that actually exist? The line `all_states =  mapper.add_geocode(all_states, "state_code", "state_id")` removes the codes that don't exist, so not a big deal.

**Comment by rumackaaron** [comment]
Looks fine to me. I made some comments but nothing needs to be changed.

File: hhs_hosp/delphi_hhs/run.py (hunk 3 above)
Comment (chinandrew) [reply hhs_hosp/delphi_hhs/run.py:36]: might be useful to have the geomap util contain useful constants

**Review by krivard — COMMENTED** [review]
Fixes in response to comments incoming

**Comment by chinandrew** [comment]
This needs to be added to python-ci yaml on line 19

**Pull request merged**